sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -68,8 +76,26 @@ export interface AnalyzeResultCommon {
 // @public
 export type AnalyzeResultOperationStatus = "notStarted" | "running" | "failed" | "succeeded";
 
+// @public
+export interface AzureBlobContentSource {
+    containerUrl: string;
+    prefix?: string;
+}
+
+// @public
+export interface AzureBlobFileListSource {
+    containerUrl: string;
+    fileList: string;
+}
+
 export { AzureKeyCredential }
 
+// @public
+export interface BeginBuildDocumentClassifierOptions extends OperationOptions, PollerOptions<DocumentClassifierOperationState> {
+    description?: string;
+    tags?: Record<string, string>;
```

Comment:
I didn't think the service let you pass tags into build classifier

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/FormRecognizer/preview/2023-02-28-preview/FormRecognizer.json#L1193

Reply:
Weird, but okay! Thanks for the catch.